themes/api.py

```diff
@@ -262,3 +263,101 @@ def update_post(self, post_id, text):
             raise ValueError('Post a url cannot be updated')
 
         return post.edit(text)
+
+    def create_comment(self, text, post_id=None, comment_id=None):
+        """
+        Create a new comment in reply to a post or comment
+
+        Args:
+            text(str): the text of the comment
+            post_id(str): the parent post id if replying to a post
+            comment_id(str): the parent comment id if replying to a comment
+
+        Raises:
+            ValueError: if both post_id and comment_id are provided
+
+        Returns:
+            praw.models.Comment: the submitted comment
+        """
+        if post_id is not None and comment_id is not None:
```

Comment:
We should also check for the case where both are `None`, which might be a common case since both arguments are optional